kount/client.py

```diff
@@ -79,7 +79,12 @@ def process(self, params):
                 logger.debug(error)
                 raise ValueError(error)
         else:
-            logger.debug("process json: %s", req_json)
+            roundtrip = request.elapsed.total_seconds()
+            merc = headers_api.get("X-Kount-Merc-Id", None)
+            sess = params.get("SESS", None)
+            logger.debug("process json: %s, MERC = %s, SESS = %s, "
```

Comment:
Remove logging of the whole request; `SESS` should be enough to track the request at Kount side.